compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -1239,15 +1245,16 @@ impl Context {
         source: BlockId,
         destination: BlockId,
         array_len: usize,
-    ) -> Result<(), RuntimeError> {
+    ) -> Result<im::Vector<AcirValue>, RuntimeError> {
```

Comment:
I imagine we were returning this for debugging? We should be able to remove this now.